proto/api/api.proto

```diff
@@ -181,6 +181,7 @@ message ActionInfo {
 message ReceiptInfo {
   iotextypes.Receipt receipt = 1;
   string blkHash = 2;
+  string executionErrorMessage = 3;
```

Comment:
it seems most straightforward to me that we just need to add `string executionError` to iotextypes.Receipt?

1. we only need to store 1 revert error message, and `iotextypes.Receipt` already contains action hash
2. and we don't need the change to transaction_log.proto as below

Reply:
but i just thought of 1 thing, that is, if we add it to iotextypes.Receipt, is the hash compatible with current protobuf, with executionError = empty?
```go
hash( newReceipt(a, b, c ... "") == hash( oldReceipt(a, b, c ...) )?
```
if not, this won't work.

Reply:
I verified this working well

Reply:
![image](https://user-images.githubusercontent.com/31757627/102046416-a4d07d00-3e1e-11eb-8621-b442f4dce69d.png)